### ISSUE TITLE
Error with non-Rancher-created containers on Rancher host

### ISSUE THREAD

**Issue by dizlexik** [disaster37/rancher-backup#18]
I'm getting the following error I think because I have containers running on a Rancher host that were not not created through Rancher itself, i.e. not part of a stack, but they are still somehow appearing in the list from getStacks.

```
2017-08-02 17:13:16,898 - DEBUG - Grab setting for stack dockertest
2017-08-02 17:13:16,899 - ERROR - Unattented error occur : 'dict' object has no attribute 'exportconfig'
2017-08-02 17:13:17,146 - ERROR - Traceback (most recent call last):
  File "/opt/backup/backup.py", line 229, in <module>
    listStacks = rancherService.getStacks()
  File "/opt/backup/fr/webcenter/backup/Rancher.py", line 83, in getStacks
    stack['settings'] = self._client.action(stack, 'exportconfig')
  File "/usr/lib/python2.7/site-packages/gdapi.py", line 400, in action
    url = getattr(obj.actions, action_name)
  File "/usr/lib/python2.7/site-packages/gdapi.py", line 112, in __getattr__
    return getattr(self.__dict__, k)
AttributeError: 'dict' object has no attribute 'exportconfig'
```

I had a separate Rancher environment where this actually was working great but now on this new one with non-Rancher containers floating around I'm running into this issue. Any ideas? Thanks!

**Comment by disaster37** [comment]
I don't really understand the problem. The problem seems appear on the export stack (can you confirm ?)
But if your stack is created directly on host (not a Rancher stack), i doesn't understand why the Rancher API know them ?

**Comment by dizlexik** [comment]
Thanks for your reply. I'm sorry but I don't have this running any longer and I don't think I'll have any time to look back into this issue because we've moved on to a different backup solution. But I do recall that while I was digging into this before I posted the issue I was pretty sure I'd tracked it down to the fact that the Rancher API was enumerating all containers on the system and not just its own managed containers, and that seemed to be why the backup process was failing with that error message. I don't think I can give you any more information at this point, but I do still think it is a legitimate bug. Let me know if you have any other questions.
